daml_dit_ddit/subcommand_build.py

```diff
@@ -48,6 +48,21 @@ def check_target_file(filename: str, force: bool):
         else:
             die(f'Target file already exists: {filename}')
 
+def file_in_zip(zip: 'ZipFile', name: str):
+    if name in zip.namelist():
+        LOG.warn(f'  File {name} exists in archive -- skipping.')
```

Comment:
This log message doesn't belong here - reason being that if you put the log message here, any call to `file_in_zip` runs the risk of printing the message, whether it's skipping the file or not.